1forge/src/endpoint/balance.ts

```diff
@@ -0,0 +1,47 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { ExecuteWithConfig, Config } from '@chainlink/types'
+import { util } from '@chainlink/ea-bootstrap'
+
+export const NAME = 'balance'
+
+const customParams = {
+  base: ['base', 'from'],
+  quote: ['quote', 'to'],
+  endpoint: false,
+  quantity: false,
+}
+
+export const execute: ExecuteWithConfig<Config> = async (request, config) => {
+  const validator = new Validator(request, customParams)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const endpoint = validator.validated.data.endpoint || 'convert'
+  const url = `https://api.1forge.com/${endpoint}`
+  const from = validator.validated.data.base.toUpperCase()
+  const to = validator.validated.data.quote.toUpperCase()
+  const quantity = validator.validated.data.quantity || 1
+  const apiKey = util.getRandomRequiredEnv('API_KEY')
```

Comment:
I think all API keys should get injected at a different point, but as some refactors will be needed leaving it out of this PR.

Reply:
Not sure I follow this...

The API key should be configured:

```suggestion
  const { apiKey } = config
```

Reply:
Yeah agreed on how the API key is configured, but what I meant is I think the changes involving getting API keys in the Config that get around `.getRandomRequiredEnv` should probably he handled in a separate issue and PR. The change will touch most EAs and I don't want the scope of this PR to grow too large.

Reply:
Handled that change in #218 and rebased into this branch to make this change. Resolving.